tests/test_strain.py

```diff
@@ -26,10 +26,16 @@ def test_hash(strain: Strain):
     assert hash(strain) == hash("strain_1")
 
 
+def test_names(strain: Strain):
+    assert strain.names == {"strain_1", "strain_1_a"}
```

Comment:
Where is the alias added? I see that in the Strain class it defaults to an empty set. 
Also for test_repr and test_str, you assert that there is 1 alias, but the length of _aliases should be 0, or am I wrong?

Reply:
The `strain` is a shared fixture defined in the `tests/conftest.py`. There is one alias for the strain with name `strain_1_a`.

Reply:
Oh now I saw it and everything makes sense :-)